ban/core/exceptions.py

```diff
@@ -1,3 +1,7 @@
+class ValidationError(ValueError):
+    ...
```

Comment:
I'm in the process of using this as much as possible to extend it's usage and make it as explicit as `pass` :p
Should I refrain? :)